x-pack/plugins/fleet/server/routes/setup/handlers.ts

```diff
@@ -46,21 +46,20 @@ export const fleetSetupHandler: RequestHandler = async (context, request, respon
   try {
     const soClient = context.core.savedObjects.client;
     const esClient = context.core.elasticsearch.client.asCurrentUser;
-    const body: PostIngestSetupResponse = await setupIngestManager(soClient, esClient);
+    const setupStatus = await setupIngestManager(soClient, esClient);
```

Comment:
No behavior change here. Simply renamed `body` to `setupStatus ` status so it could be a distinct `SetupStatus` type (has `Error` objects in array)